sdk/src/ssi/did.go

```diff
@@ -21,23 +22,24 @@ type DIDKeyWrapper struct {
 // // where secp is an import alias to the secp256k1 library we use "github.com/decred/dcrd/dcrec/secp256k1/v4"
 // secpPrivKey, ok := privKey.(secp.PrivateKey)
 // if !ok { ... }
-func GenerateDIDKey(kt string) (*DIDKeyWrapper, error) {
+func GenerateDIDKey(kt string) ([]byte, error) {
```

Comment:
Could you update the godoc for this function so it's clear what this returns? 

Something like the following.

```go
// GenerateDIDKey takes in a key type value that this library supports and constructs a conformant did:key identifier.
// The function returns the marshaled JSON representation of `generateDIDKeyResult`.
```